### PR TITLE
Add InigoMCM

### PR DESCRIPTION
**Pull request by pStyl3**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -907,6 +907,17 @@ plugins:
         util: '[TES5Edit v4.0.4b](https://www.nexusmods.com/skyrim/mods/25859)'
         itm: 36
 
+  - name: 'InigoMCM.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrim/mods/70035/'
+        name: 'InigoMCM'
+    req:
+      - *SKSE
+      - *SkyUI
```
Comment (MacSplody) [inline masterlist.yaml:916]: SkyUI requires SKSE, so you don't need to specify both as requirements.

```yaml
    req: [ *SkyUI ]
```

Comment (pStyl3) [reply masterlist.yaml:916]: While that may be true, stating both of them has been done for multiple plugins within the SLE masterlist, so I followed that as usual.

Comment (pStyl3) [reply masterlist.yaml:916]: Ah I see, the SSE masterlist doesn't feature the `&SkyUI` anchor, but rather it's done like so:
```yaml
  - name:  'StormLightning.esp'
    msg: [ *requiresMCM ]
    req: [ *SKSE ]
```
with
```yaml
  - &requiresMCM
    <<: *requiresMCM_X
    subs: [ '[SkyUI SE](https://www.nexusmods.com/skyrimspecialedition/mods/12604/)' ]
    condition: 'not active("SkyUI_SE.esp")'
```

Comment (MacSplody) [reply masterlist.yaml:916]: It used to have that anchor, but there are very few mods that explicitly require SkyUI to function, so it was removed.

Comment (pStyl3) [reply masterlist.yaml:916]: We can update the SLE masterlist to reflect those changes in the SSE masterlist, no problem from my side.

**Pull request merged**